src/main/scala/org/broadinstitute/dsde/firecloud/webservice/UserApiService.scala

```diff
@@ -182,6 +183,11 @@ trait UserApiService extends HttpService with PerRequestCreator with FireCloudRe
           }
         }
       }
+      pathPrefix("proxyGroup") {
+        path(Segment) { email =>
+          passthrough(UserApiService.samUserProxyGroupURL(email), HttpMethods.GET)
```

Comment:
Should this route segment be wrapped in a `get {...}` block?

Reply:
Good question. Should it? I've seen passthroughs wrapped before but that looked to me like a holdover from the implementations previously not being passthroughs. I was thinking that specifying `HttpMethods.GET` in the call to `passthrough` was enough to only apply to that method, but I'm glad you asked.

Can anyone confirm how this should be?

Reply:
What I think will happen here is that all methods (`PUT`, `POST`, etc.) will be handled and channeled to a GET. It will work, but doesn't exactly follow the swagger spec.

Reply:
My understanding is that this would already reject anything that's not a `GET`.

Since we have some confusion on this, would you mind testing this assumption and adding a comment to the `passthrough()` method itself based on what you find?

Reply:
Ha! Tests! Yeah, I should add some!

I will also clarify the behavior of `passthrough` as you suggested. Thanks.